### ISSUE TITLE
[Raid - Zul'Gurub] Hakkar not working like in retail

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5805]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Hakkar mind controls even if there is only one person in the raid instance and then resets his HP to full, unlike in retail WoTLK. In retail WoTLK I never had him reset HP to full during this encounter, even having priests left unkilled (however, in this raid I killed all the priests). I've tried solo with a hunter with the pet as top threat, and I've tried solo as a ret paladin.
**Expected behaviour**: Tell us what should happen instead
Hakkar should not mind control and reset to 100% HP when there is only 1 person in the raid instance, or this boss never would have been soloable in retail WoTLK.
**Steps to reproduce the problem**:

1. Try to kill Hakkar solo.
2. Get mind controlled.
3. Hakkar's HP resets, smash head against wall because this is not how it worked in retail.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
https://wotlk-twinhead.twinstar.cz/?npc=14834
Raid ID 1202
You can research this on several pages like
https://wowwiki.fandom.com/wiki/Hakkar_the_Soulflayer_(tactics)

"If you are the only one fighting Hakkar he wont use his MC ability. If Bloodworms or a ghoul is active, he WILL use his MC, and you will kill it(them) before the duration of the MC is up, thus causing him to reset, and fully heal."

**Include Screenshots from the issue if necessary**
![WoWScrnShot_081119_175816](https://user-images.githubusercontent.com/50332138/62841472-aa83e480-bc65-11e9-860b-14501342a0d6.jpg)

**Comment by Lothloryen** [comment]
Hello there,

Thank you very much for the report. This issue has been fixed and it will be working as it should after the next scheduled maintenance.
